kf_lib_data_ingest/etl/transform/transform.py

```diff
@@ -44,10 +44,11 @@ def _read_output(self):
         """
         Read previously written transform stage output
 
-        :returns: dict of pandas.DataFrames
+        :returns: dict of pandas.DataFrames keyed by strings
+        representing target concepts (i.e. participant, biospecimen, etc)
         """
         output = {
-            filename.split('.')[0]: pandas.read_csv(
+            os.path.splitext(filename)[0]: pandas.read_csv(
```

Comment:
What's the reason for doing this splitexit?

Reply:
I'm just trying to reproduce the same output that TransformStage._run produces. Once its fully implemented, TransformStage._run will return a dict of dataframes, where the keys are target concepts. So it will be like this:

**Run stage**
```
output = TransformStage.run(mapped_df_dict)
# Output looks like this:
{
    'participant': df_p,
    'biospecimen': df_b,
    'genomic_file': df_g
}
``` 
**Written output looks like this**
```
test_study/output/TransformStage/participant.tsv
test_study/output/TransformStage/biospecimen.tsv
test_study/output/TransformStage/genomic_file.tsv
```
**Read previously written output files and reproduce original output data structure**
```
Now see TransformStage._read_output
```

Reply:
This may possibly change, not sure yet. I need to implement the rest of transform first

Reply:
oh I see